integrationTests/ts/__tests__/integration.test.ts

```diff
@@ -15,7 +15,7 @@ import {
   DeployedContracts,
   PollContracts,
 } from "maci-cli";
-import { join } from "path";
+import path from "path";
```

Comment:
Seeing that you changed here to import the full library, maybe let's do the same for `fs` as you did in other files (in general I saw that when you updated path you also updated fs)? would be great to be consistent across the repo (can also tackle in a separate PR if you'd prefer)

Reply:
Yes, for all the packages I do it step by step. At the end all the codebase will be consistent.